microsoft-edge/devtools-guide-chromium/whats-new/2023/12/devtools-120.md

```diff
@@ -0,0 +1,103 @@
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 12/07/2023
+---
+# What's New in DevTools (Microsoft Edge 120)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## DevTools has a new user interface
+
+<!-- Subtitle: Introducing the all-new Edge DevTools user interface.  Check out our blog post to learn about the new Activity Bar, Quick View, and how to customize your developer tools in the redesigned DevTools UI. -->
+<!-- todo: the above is from Welcome 119 -->
+
+DevTools has a new look, starting with Microsoft Edge 120!  The major design updates are the new **Activity Bar** with icons; **Quick View** in place of the Drawer at bottom of DevTools; and the ability to easily open, close, and move tools around:
+
+![Updated DevTools UI](./devtools-120-images/new-devtools-ui.png)
+
+See blog post: [Inside the all-new Edge DevTools user interface](https://blogs.windows.com/msedgedev/2023/12/20/inside-the-all-new-edge-devtools-ui/).
+
+If you prefer the legacy UI, you can switch back by going to DevTools > **Settings** > **Preferences** and selecting the **Switch back to the legacy DevTools UI** checkbox.  After reloading DevTools, the legacy UI appears.
+
+
+<!-- ====================================================================== -->
+## The "Explain with Copilot in DevTools" feature is generally available
+
+<!-- Subtitle: Debug source code by having Copilot explain lines of code.  You can ask Copilot to read lines of code for you and describe what the code is doing for an easier time debugging. -->
+<!-- todo: the above is from Welcome 119 -->
+
+You can now ask DevTools to explain any Console error, Console warning, or code snippet by using Copilot in Edge:
+
+![Explain with Copilot in DevTools](./devtools-120-images/copilot-error-explanation.png)
+
+This feature is enabled by default, starting with Microsoft Edge 120.  See blog post: [Navigate the unexpected: using Copilot in Edge and DevTools](https://blogs.windows.com/msedgedev/2023/12/14/navigate-the-unexpected-using-copilot-in-edge-and-devtools/).
+
+If you don't see the feature, you might need to enable it.  To enable this feature, follow the steps in [Enable the features](../../../experimental-features/copilot-explain.md#enable-the-features) in _Explain DevTools Console errors and source code using Copilot in Edge_.
+
+If you have any questions, comments, or feedback about the feature, please leave a comment in the GitHub Issue: [\[Feedback\]\[Experiment\] Explain errors, warnings, and source code in the Console and Sources tools](https://github.com/MicrosoftEdge/DevTools/issues/203).
+
+
+<!-- ====================================================================== -->
+## New Recorder tool to record and replay user flows and measure performance
+
+<!-- Subtitle: todo. -->
```

Comment:
@leahmsft @zoherghadyali - need Subtitle for What's New 120 entry "New Recorder tool to record and replay user flows and measure performance"